packages/components/src/interfaces/complex/Image.ts

```diff
@@ -20,11 +20,21 @@ export const generateImageSrcSchema = ({
 
 export const ImageSrcSchema = generateImageSrcSchema({})
 
+// Note: ajv pattern does not support the use of patternFlag like "i" for case-insensitive
+// Thus, we manually add the case-insensitive flag to the regex pattern
+export const ALT_TEXT_REGEX_PATTERN =
```

Comment:
[nit] can i trouble you to add 1 line above this to explain what the regex does?